docs/src/use/getting-started.md

```diff
@@ -58,43 +58,19 @@ yarn run eslint yourfile.js
 
 **Note:** If you are coming from a version before 9.0.0 please see the [migration guide](configure/migration-guide).
 
-After running `npm init @eslint/config`, you'll have an `eslint.config.js` (or `eslint.config.mjs`) file in your directory. In it, you'll see some rules configured like this:
+When you run `npm init @eslint/config`, terminal would provide you different options to create a config file `eslint.config.js` (or `eslint.config.mjs`) as per your need.
 
-```js
-// eslint.config.js
-export default [
-    {
-        rules: {
-            "no-unused-vars": "error",
-            "no-undef": "error"
-        }
-    }
-];
-```
-
-The names `"no-unused-vars"` and `"no-undef"` are the names of [rules](../rules) in ESLint. The first value is the error level of the rule and can be one of these values:
-
-* `"off"` or `0` - turn the rule off
-* `"warn"` or `1` - turn the rule on as a warning (doesn't affect exit code)
-* `"error"` or `2` - turn the rule on as an error (exit code will be 1)
-
-The three error levels allow you fine-grained control over how ESLint applies rules (for more configuration options and details, see the [configuration docs](configure/)).
-
-Your `eslint.config.js` configuration file will also include a recommended configuration, like this:
+For example if you select *browser` and choose to install the `dependencies`, you will get a config like the one shown below.
```

Comment:
For example, one of the questions is "Where does your code run?" If you select "Browser" then your configuration file will contain the definitions for global variables found in web browsers. Here's an example: